### ISSUE TITLE
Fix how `Dataframe` examples look when a string is passed in

### ISSUE THREAD

**Issue by abidlabs** [gradio-app/gradio#2092]
### Describe the bug

With the latest version of Gradio, you can pass in string filepaths to CSVs as examples to the `Dataframe` component. However, the examples are not rendered properly in the Examples component, as you can see here:

<img width="605" alt="image" src="https://user-images.githubusercontent.com/1778297/186937403-adcab66e-e0aa-4737-a2c5-a4e41f278ce6.png">

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

```py
import gradio as gr
import pandas as pd
import numpy as np

examples = [["test.csv"]]

def get_max(df: pd.DataFrame) -> float:
    return df.max().max()


gr.Interface(get_max, gr.Data
Frame(type="pandas", headers=["a", "b", "c"]), gr.Number(),
             examples=examples).launch()
```

Also have the attached `test.csv` file in your directory.

[test.csv](https://github.com/gradio-app/gradio/files/9434365/test.csv)

### Logs

```shell
N/A
```

### System Info

```shell
latest
```

### Severity

annoying

**Comment by merveenoyan** [comment]
After I made it a list of 2D lists it worked, but still it was looking like strings with gaps instead of structured data. See below :') Maybe we could warn user.
<img width="776" alt="Ekran Resmi 2022-08-26 20 35 41" src="https://user-images.githubusercontent.com/53175384/186969943-71277f2a-a0d8-4936-bb26-a4e948f5f786.png">

**Comment by abidlabs** [comment]
Agreed, in general examples for Dataframes could look nicer. Let me broaden the issue
